elements/pf-select/pf-select-option.css

```diff
@@ -0,0 +1,60 @@
+:host {
+  display: block;
+  padding: var(--pf-global--spacer--sm, 0.5rem) var(--pf-global--spacer--md, 1rem);
+}
+
+:host([hidden]),
+:host([hidden-by-filter]),
+*[hidden] {
+  display: none !important;
+}
+
+:host(:focus),
+:host(:hover) {
+  background-color: #e0e0e0;
+}
+
+:host([active-descendant]) {
+  background-color: var(--_pf-select-option-active-descendant-color, var(--pf-theme--color--surface--lighter, #f0f0f0));
+}
+
+:host([disabled]) {
+  color: var(--pf-global--Color--dark-200, #6a6e73);
+  pointer-events: none;
+}
+
+div {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+input[type="checkbox"] {
+  margin-inline-end: 1em;
+  display: var(--_pf-select-option-checkboxes-display, none);
+  pointer-events: none;
+  flex: 0 0 auto;
+}
+
+span {
+  flex: 1 1 auto;
+}
+
+svg {
+  color: var(--_pf-select-option-svg-color, var(--pf-theme--color--accent, #0066cc));
+  width: 1em;
+  height: 1em;
+  margin-inline-start: 1em;
+  text-align: right;
+  flex: 0 0 auto;
+}
```

Comment:
SVG check icon in PFv4 is displayed as 10px.  They have a span that applies this font-size wrapping it.  We aren't doing the same positioning technique but assume we'd still want the 10px size icon.
```Suggestion
svg {
  font-size: var(--pf-c-select__menu-item-icon--FontSize, var(--pf-global--icon--FontSize--sm, 0.675rem));
  color: var(--_pf-select-option-svg-color, var(--pf-theme--color--accent, #0066cc));
  width: 1em;
  height: 1em;
  margin-inline-start: 1em;
  text-align: right;
  flex: 0 0 auto;
}
```